ql/src/java/org/apache/hadoop/hive/ql/io/orc/encoded/EncodedTreeReaderFactory.java

```diff
@@ -298,11 +300,11 @@ public void seek(PositionProvider index) throws IOException {
       }
     }
 
-    @Override
+    @Override        
     public void nextVector(
-        ColumnVector previousVector, boolean[] isNull, int batchSize) throws IOException {
+        ColumnVector previousVector, boolean[] isNull, int batchSize, FilterContext filterContext, TypeReader.ReadPhase readPhase) throws IOException {
```

Comment:
this line looks a bit long :) can you reformat according to [this](https://github.com/apache/hive/blob/master/dev-support/eclipse-styles.xml)?

Reply:
Reformatted to match the style.